src/com/goide/ui/ProjectTutorialNotification.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2013-2015 Sergey Ignatov, Alexander Zolotov, Mihai Toader, Florin Patan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.goide.ui;
+
+import com.intellij.ide.util.PropertiesComponent;
+import com.intellij.notification.*;
+import com.intellij.openapi.components.*;
+import com.intellij.util.text.VersionComparatorUtil;
+import org.jetbrains.annotations.NotNull;
+
+import static com.goide.util.GoUtil.PLUGIN_VERSION;
+
+public class ProjectTutorialNotification implements ApplicationComponent {
+
+  private static final String GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN = "go.tutorial.project.notification.shown";
+
+  @Override
+  public void initComponent() {
+    final PropertiesComponent propertiesComponent = PropertiesComponent.getInstance();
+    boolean shownAlready;
+    //noinspection SynchronizationOnLocalVariableOrMethodParameter
+    synchronized (propertiesComponent) {
+      String storedVersion = propertiesComponent.getValue(GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN, "0.0");
+      shownAlready = VersionComparatorUtil.compare(storedVersion, PLUGIN_VERSION) == 0;
+
+      if (!shownAlready) {
+        propertiesComponent.setValue(GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN, PLUGIN_VERSION);
+      }
+    }
+    
+    if (shownAlready) {
+      return;
+    }
+
+    NotificationGroup group = NotificationGroup.findRegisteredGroup("Plugins Suggestion");
```

Comment:
Still don't understand why do you need group. Why not just

```
Notifications.Bus.notify(new Notification());
```

Group will be found by its name or created.

'Plugins Suggestion' is not the best name, it's unlikely that user will be able to associate it with Go Plugin. Use 'Go'-group (it's used in quick fixes) or make the new one but with 'Go' word.